tests/Dfe.PlanTech.Web.E2ETests/cypress/commands/login.js

```diff
@@ -10,6 +10,8 @@ Cypress.Commands.add("login", ({ email, password, url }) => {
     
   cy.visit(url);
   cy.get("input#username").type(email);
+  cy.get("div.govuk-button-group button.govuk-button").first().click();
+  cy.wait(2000);
```

Comment:
400 for a longer wait time to match the password maybe? just so you're not hamstrung if it's slow when running the pipeline

Reply:
[313e537](https://github.com/DFE-Digital/sts-plan-technology-for-your-school/pull/731/commits/313e5370e11c45046b6ec0b2d2ee696357a4f898)

Reply:
IMO it would be better to amend the line below to wait longer.

E.g.

```js
cy.get("input#password", { timeout: 2000 }).type(password);
```

Which would mean that it would wait 2 second for thd the `input#password` element to exist, before failing.

This way:
1. If it shows up earlier then the timeout ends - which will result in quicker tests more often than not
2. it is more explicit about what you're waiting for - you're waiting for that specific element to show.